Gemfile.lock

```diff
@@ -464,6 +464,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86_64-darwin-18
```

Comment:
why do we need this?

Reply:
my laptop added it automatically when I did a bundle install, havent' yet looked into why though

Reply:
seems like it occurs for others as well